### DISCUSSION TITLE
Exclude own views by checking localstorage

### DISCUSSION THREAD

**Discussion by dimitrisnl** [plausible/analytics#513] [Anything Else]
Hey there. I just connected my website with Plausible and I'm very impressed!

I was wondering if the process of excluding my own views could be improved. [Fathom uses a local storage flag](https://usefathom.com/support/exclude) in order to hold back from tracking. Would this be viable? 

Thanks!

**Comment by metmarkosaric** [comment]
Thanks @dimitrisnl! there's a pull request to do something like that already so should be completed soon https://github.com/plausible/analytics/pull/489

**Comment by dimitrisnl** [comment]
That's fantastic @metmarkosaric!

Is there a way to mark this discussion as closed or answered?

**Reply by metmarkosaric** [reply]
Great! we're using the new GitHub Discussions so it can stay here if others are having a similar question.

**Comment by metmarkosaric** [comment]
This is now live! See https://plausible.io/docs/excluding
